data/eos-autoupdater.conf

```diff
@@ -10,4 +10,5 @@
 #  3 - apply updates
 LastAutomaticStep=3
 IntervalDays=14
+RandomizedDelayDays=7
```

Comment:
Hmm... I'm not sure if the Random delay is so bad - we "hope" that we distribute our updates over a 2 week period, but we don't actually do anything to ensure this or counteract factors like computers which are off during the week, or off during the weekend, which will cause clustering. Generally, configuration changes between the default and the deployments introduce risk, because it's code paths we will inevitably test less. Is there anything to worry about having this switched on usually?